Build Issue with the latest teeio-validator

Hi !

We are seeing build errors while running "make -j" command after pulling latest updates to teeio-validator.

Error Message:

 77%] Built target pci_ide_km_requester_lib
[ 77%] Built target spdm_common_lib
[ 77%] Built target pci_tdisp_requester_lib
[ 77%] Built target cryptlib_mbedtls
/home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/teeio_validator/teeio_debug.c: In function ‘teeio_assert’:
/home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/teeio_validator/teeio_debug.c:61:6: error: infinite recursion detected [-Werror=infinite-recursion]
   61 | void teeio_assert(const char *file_name, int line_number, const char *description)
      |      ^~~~~~~~~~~~
In file included from /home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/include/utils.h:15,
                 from /home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/teeio_validator/teeio_debug.c:12:
/home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/include/teeio_debug.h:31:13: note: recursive call
   31 |             teeio_assert(__FILE__, __LINE__, #expression); \
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/teeio_validator/teeio_debug.c: In function ‘teeio_assert’:
/home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/teeio_validator/teeio_debug.c:61:6: error: infinite recursion detected [-Werror=infinite-recursion]
   61 | void teeio_assert(const char *file_name, int line_number, const char *description)
      |      ^~~~~~~~~~~~
In file included from /home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/include/utils.h:15,
                 from /home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/teeio_validator/teeio_debug.c:12:
/home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/include/teeio_debug.h:31:13: note: recursive call
   31 |             teeio_assert(__FILE__, __LINE__, #expression); \
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
cc1: all warnings being treated as errors
make[2]: *** [teeio_validator/CMakeFiles/lside.dir/build.make:104: teeio_validator/CMakeFiles/lside.dir/teeio_debug.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [teeio_validator/CMakeFiles/setide.dir/build.make:104: teeio_validator/CMakeFiles/setide.dir/teeio_debug.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:1196: teeio_validator/CMakeFiles/setide.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:1170: teeio_validator/CMakeFiles/lside.dir/all] Error 2
[ 77%] Built target spdm_crypt_lib
[ 77%] Built target spdm_requester_lib
[ 77%] Built target mbedcrypto
make: *** [Makefile:136: all] Error 2
root@nte-gnr-cxl:/home/nte/Documents/tee_io_val/tee-io-validator/teeio-validator/build#

--- a/teeio-validator/teeio_validator/teeio_debug.c
+++ b/teeio-validator/teeio_validator/teeio_debug.c
@@ -78,6 +78,6 @@ void teeio_assert(const char *file_name, int line_number, const char *descriptio
 #endif
 #endif
 
-    TEEIO_ASSERT(false);
+    assert(false);
 }
 #endif /* TEEIO_ASSERT_ENABLE */

